Source/WebCore/page/Quirks.cpp

```diff
@@ -1023,7 +1023,8 @@ void Quirks::triggerOptionalStorageAccessIframeQuirk(const URL& frameURL, Comple
                 return;
             }
         }
-        if (subFrameDomainsForStorageAccessQuirk().contains(RegistrableDomain { frameURL })) {
+        bool isMSOLoginButNotMSTeams = document->url().hasQuery() && document->url().hoat() == "login.microsoftonline.com"_s && !document->url().query().contains("redirect_uri=https%3A%2F%2Fteams.microsoft.com"_s);
```

Comment:
`hoat()` -> `host()`

Reply:
Sorry, I didn't see this comment before adding the merge label. I removed it again. Hopefully it doesn't land.

Reply:
I added the `merging-blocked` keyword which I think is the way to make sure.

Reply:
I fixed the typo and uploaded a revised patch before you cq+'d it, so all is good!